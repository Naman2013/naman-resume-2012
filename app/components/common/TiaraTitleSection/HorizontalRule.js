import React, { Fragment } from 'react';
import PropTypes from 'prop-types';

const HR = ({ theme }) => (
  <Fragment>
    <hr style={theme} />
    <style jsx>
      {`
        hr {
          height: 0;
          border: 0;
          margin: 0;
          padding: 0;
          position: relative;
        }
      `}
    </style>
  </Fragment>
);

HR.propTypes = { theme: PropTypes.shape({}) };
HR.defaultProps = { theme: {} };

export default HR;
