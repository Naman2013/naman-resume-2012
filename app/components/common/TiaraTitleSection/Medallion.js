import React from 'react';
import PropTypes from 'prop-types';
import { golda, blue_tile_feat } from '../../../styles/variables/colors_tiles_v4';

function calculateIconSize(dimension) {
  const iconToDimensionRatio = 0.5;
  return (dimension * iconToDimensionRatio);
}

const Medallion = ({ iconURL, dimension, theme }) => (
  <div
    style={theme}
  >
    <img
      alt=""
      width={calculateIconSize(dimension)}
      height={calculateIconSize(dimension)}
      src={iconURL}
    />
    <style jsx>
      {`
        div {
          position: relative;
          width: ${dimension}px;
          height: ${dimension}px;
          border-radius: 50%;
          border: 5px solid ${golda};
          background: url(${blue_tile_feat});
          margin: 0;
          padding: 0;
        }

        img {
          position: absolute;
          top: 50%;
          left: 50%;
          transform: translate(-50%, -50%);
        }
      `}
    </style>
  </div>
);

Medallion.propTypes = {
  iconURL: PropTypes.string.isRequired,
  theme: PropTypes.shape({}),
  dimension: PropTypes.number,
};

Medallion.defaultProps = {
  theme: {},
  dimension: 50,
};

export default Medallion;
