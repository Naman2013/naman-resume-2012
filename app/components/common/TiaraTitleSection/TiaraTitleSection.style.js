import css from 'styled-jsx/css';
import { primaryFont, secondaryFont } from '../../../styles/variables/fonts';
import {
  astronaut,
  blue_tile_feat,
  white_tile_paper,
  midnight_express,
  golda,
  golden,
} from '../../../styles/variables/colors_tiles_v4';
import {
  screenMedium,
  screenLarge,
  screenXLarge,
} from '../../../styles/variables/breakpoints';

export default css`
  .root {
    position: relative;
    text-align: center;
    font-family: ${primaryFont};
    color: ${astronaut};
    background: url(${blue_tile_feat});
    padding-top: 70px;
  }

  .center-line {
    position: absolute;
    top: 0;
    z-index: 5;
    width: 100%;
    height: 100%;
    width: 50%;
    border-right: 2px solid ${astronaut};
  }

  .title {
    padding-top: 80px;
    padding-bottom: 60px;
    font-weight: 300;
    text-transform: capitalize;
    font-size: 22px;
    font-family: ${secondaryFont};
  }

  .pre-title {
    display: block;
    text-transform: uppercase;
    font-weight: bold;
    letter-spacing: 2px;
    font-size: 10px;
    margin-bottom: 20px;
    font-family: ${primaryFont};
  }

  @media ${screenMedium} {
    .root {
      padding-top: 120px;
    }

    .title {
      font-size: 36px;
    }
  }

  @media ${screenLarge} {
    .title {
      font-size: 44px;
      padding-bottom: 120px;
    }

    .pre-title {
      margin-bottom: 50px;
    }
  }

  @media ${screenXLarge} {
    .root {
      padding-top: 160px;
    }

    .title {
      font-size: 42px;
    }
  }
`;
