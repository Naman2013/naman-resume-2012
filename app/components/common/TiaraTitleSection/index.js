import TiaraTitleSection from './TiaraTitleSection';

export default TiaraTitleSection;
