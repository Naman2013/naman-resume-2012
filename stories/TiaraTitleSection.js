import React from 'react';
import { storiesOf } from '@storybook/react';
import TiaraTitleSection from '../app/components/common/TiaraTitleSection';

storiesOf('TiaraTitleSection', module)
  .add('Mobile', () => (
    <TiaraTitleSection />
  ));
