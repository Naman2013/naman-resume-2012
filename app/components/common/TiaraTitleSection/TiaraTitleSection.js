import React from 'react';
import PropTypes from 'prop-types';
import Medallion from './Medallion';
import { primaryFont } from '../../../styles/variables/fonts';
import { astronaut } from '../../../styles/variables/colors_tiles_v4';

const TiaraTitleSection = ({ preTitle, title, iconURL }) => (
  <div className="root">
    <Medallion iconURL={iconURL} />
    <h2 className="title">
      <span className="pre-title">{preTitle}</span>
      {title}
    </h2>

    <style jsx>
      {`
        .root {
          text-align: center;
          font-family: ${primaryFont};
          color: ${astronaut};
        }

        .title {
          font-weight: 300;
          text-transform: capitalize;
        }

        .pre-title {
          display: block;
          text-transform: uppercase;
          font-weight: bold;
          letter-spacing: 2px;
          font-size: 10px;
          margin-bottom: 20px;
        }
      `}
    </style>
  </div>
);

TiaraTitleSection.defaultProps = {
  preTitle: '',
  iconURL: '',
};

TiaraTitleSection.propTypes = {
  preTitle: PropTypes.string,
  iconURL: PropTypes.string,
  title: PropTypes.string.isRequired,
};

export default TiaraTitleSection;
