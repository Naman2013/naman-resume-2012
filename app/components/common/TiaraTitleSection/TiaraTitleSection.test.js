import React from 'react';
import { shallow } from 'enzyme';
import TiaraTitleSection from './TiaraTitleSection';
import alienHeadIcon from '../../../../assets/icons/alien-head.png';

const setup = (customProps = {}) => {
  const defaultProps = { preTitle: 'Test', title: 'Test', iconURL: alienHeadIcon };
  const props = Object.assign({}, defaultProps, customProps);
  const shallowWrapper = shallow(<TiaraTitleSection {...props} />);

  return {
    shallowWrapper,
  };
};

describe('TiaraTitleSection', () => {
  const defaultProps = {
    iconURL: alienHeadIcon,
    preTitle: 'A guide to',
    title: 'The solar system',
  };

  it('should render correctly', () => {
    expect(setup(defaultProps).shallowWrapper).toMatchSnapshot();
  });
});
