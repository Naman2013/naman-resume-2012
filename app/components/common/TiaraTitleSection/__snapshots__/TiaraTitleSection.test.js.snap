// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TiaraTitleSection should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TiaraTitleSection
    iconURL="foo.com"
    preTitle="A guide to"
    title="The solar system"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="jsx-1727865174 center-line"
        />,
        <CenterColumn
          breakpoints={
            Array [
              "screen and (min-width: 768px)",
              "screen and (min-width: 1024px)",
              "screen and (min-width: 1366px)",
            ]
          }
          theme={
            Object {
              "background": "url(https://vega.slooh.com/assets/v4/common/White_Paper_Subtle.png)",
              "boxShadow": "rgb(191, 191, 191) 0px 11px 20px -10px",
              "position": "relative",
              "zIndex": "10",
            }
          }
          widths={
            Array [
              "620px",
              "780px",
              "940px",
            ]
          }
        >
          <Medallion
            dimension={70}
            iconURL="foo.com"
            theme={
              Object {
                "left": "50%",
                "position": "absolute",
                "top": "-30px",
                "transform": "translateX(-50%)",
                "zIndex": "20",
              }
            }
          />
          <HR
            theme={
              Object {
                "borderTop": "2px solid #c2a068",
                "top": "6px",
              }
            }
          />
          <HR
            theme={
              Object {
                "borderTop": "1px solid #e2b979",
                "top": "9px",
              }
            }
          />
          <h2
            className="jsx-1727865174 title"
          >
            <span
              className="jsx-1727865174 pre-title"
            >
              A guide to
            </span>
            The solar system
          </h2>
        </CenterColumn>,
        <JSXStyle
          css={
            Array [
              ".root.jsx-1727865174{position:relative;text-align:center;font-family:brandon-grotesque, sans-serif;color:#415671;background:url(https://vega.slooh.com/assets/v4/common/Dark_Pattern_Featured_LargeCanvas.png);padding-top:70px;}",
              ".center-line.jsx-1727865174{position:absolute;top:0;z-index:5;width:100%;height:100%;width:50%;border-right:2px solid #415671;}",
              ".title.jsx-1727865174{padding-top:80px;padding-bottom:60px;font-weight:300;text-transform:capitalize;font-size:22px;font-family:adobe-garamond-pro, serif;}",
              ".pre-title.jsx-1727865174{display:block;text-transform:uppercase;font-weight:bold;-webkit-letter-spacing:2px;-moz-letter-spacing:2px;-ms-letter-spacing:2px;letter-spacing:2px;font-size:10px;margin-bottom:20px;font-family:brandon-grotesque, sans-serif;}",
              "@media screen and (min-width: 768px){.root.jsx-1727865174{padding-top:120px;}.title.jsx-1727865174{font-size:36px;}}",
              "@media screen and (min-width: 1024px){.title.jsx-1727865174{font-size:44px;padding-bottom:120px;}.pre-title.jsx-1727865174{margin-bottom:50px;}}",
              "@media screen and (min-width: 1366px){.root.jsx-1727865174{padding-top:160px;}.title.jsx-1727865174{font-size:42px;}}",
            ]
          }
          styleId="1727865174"
        />,
      ],
      "className": "jsx-1727865174 root",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "jsx-1727865174 center-line",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "breakpoints": Array [
            "screen and (min-width: 768px)",
            "screen and (min-width: 1024px)",
            "screen and (min-width: 1366px)",
          ],
          "children": Array [
            <Medallion
              dimension={70}
              iconURL="foo.com"
              theme={
                Object {
                  "left": "50%",
                  "position": "absolute",
                  "top": "-30px",
                  "transform": "translateX(-50%)",
                  "zIndex": "20",
                }
              }
            />,
            <HR
              theme={
                Object {
                  "borderTop": "2px solid #c2a068",
                  "top": "6px",
                }
              }
            />,
            <HR
              theme={
                Object {
                  "borderTop": "1px solid #e2b979",
                  "top": "9px",
                }
              }
            />,
            <h2
              className="jsx-1727865174 title"
            >
              <span
                className="jsx-1727865174 pre-title"
              >
                A guide to
              </span>
              The solar system
            </h2>,
          ],
          "theme": Object {
            "background": "url(https://vega.slooh.com/assets/v4/common/White_Paper_Subtle.png)",
            "boxShadow": "rgb(191, 191, 191) 0px 11px 20px -10px",
            "position": "relative",
            "zIndex": "10",
          },
          "widths": Array [
            "620px",
            "780px",
            "940px",
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "dimension": 70,
              "iconURL": "foo.com",
              "theme": Object {
                "left": "50%",
                "position": "absolute",
                "top": "-30px",
                "transform": "translateX(-50%)",
                "zIndex": "20",
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "theme": Object {
                "borderTop": "2px solid #c2a068",
                "top": "6px",
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "theme": Object {
                "borderTop": "1px solid #e2b979",
                "top": "9px",
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <span
                  className="jsx-1727865174 pre-title"
                >
                  A guide to
                </span>,
                "The solar system",
              ],
              "className": "jsx-1727865174 title",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "A guide to",
                  "className": "jsx-1727865174 pre-title",
                },
                "ref": null,
                "rendered": "A guide to",
                "type": "span",
              },
              "The solar system",
            ],
            "type": "h2",
          },
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "css": Array [
            ".root.jsx-1727865174{position:relative;text-align:center;font-family:brandon-grotesque, sans-serif;color:#415671;background:url(https://vega.slooh.com/assets/v4/common/Dark_Pattern_Featured_LargeCanvas.png);padding-top:70px;}",
            ".center-line.jsx-1727865174{position:absolute;top:0;z-index:5;width:100%;height:100%;width:50%;border-right:2px solid #415671;}",
            ".title.jsx-1727865174{padding-top:80px;padding-bottom:60px;font-weight:300;text-transform:capitalize;font-size:22px;font-family:adobe-garamond-pro, serif;}",
            ".pre-title.jsx-1727865174{display:block;text-transform:uppercase;font-weight:bold;-webkit-letter-spacing:2px;-moz-letter-spacing:2px;-ms-letter-spacing:2px;letter-spacing:2px;font-size:10px;margin-bottom:20px;font-family:brandon-grotesque, sans-serif;}",
            "@media screen and (min-width: 768px){.root.jsx-1727865174{padding-top:120px;}.title.jsx-1727865174{font-size:36px;}}",
            "@media screen and (min-width: 1024px){.title.jsx-1727865174{font-size:44px;padding-bottom:120px;}.pre-title.jsx-1727865174{margin-bottom:50px;}}",
            "@media screen and (min-width: 1366px){.root.jsx-1727865174{padding-top:160px;}.title.jsx-1727865174{font-size:42px;}}",
          ],
          "styleId": "1727865174",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="jsx-1727865174 center-line"
          />,
          <CenterColumn
            breakpoints={
              Array [
                "screen and (min-width: 768px)",
                "screen and (min-width: 1024px)",
                "screen and (min-width: 1366px)",
              ]
            }
            theme={
              Object {
                "background": "url(https://vega.slooh.com/assets/v4/common/White_Paper_Subtle.png)",
                "boxShadow": "rgb(191, 191, 191) 0px 11px 20px -10px",
                "position": "relative",
                "zIndex": "10",
              }
            }
            widths={
              Array [
                "620px",
                "780px",
                "940px",
              ]
            }
          >
            <Medallion
              dimension={70}
              iconURL="foo.com"
              theme={
                Object {
                  "left": "50%",
                  "position": "absolute",
                  "top": "-30px",
                  "transform": "translateX(-50%)",
                  "zIndex": "20",
                }
              }
            />
            <HR
              theme={
                Object {
                  "borderTop": "2px solid #c2a068",
                  "top": "6px",
                }
              }
            />
            <HR
              theme={
                Object {
                  "borderTop": "1px solid #e2b979",
                  "top": "9px",
                }
              }
            />
            <h2
              className="jsx-1727865174 title"
            >
              <span
                className="jsx-1727865174 pre-title"
              >
                A guide to
              </span>
              The solar system
            </h2>
          </CenterColumn>,
          <JSXStyle
            css={
              Array [
                ".root.jsx-1727865174{position:relative;text-align:center;font-family:brandon-grotesque, sans-serif;color:#415671;background:url(https://vega.slooh.com/assets/v4/common/Dark_Pattern_Featured_LargeCanvas.png);padding-top:70px;}",
                ".center-line.jsx-1727865174{position:absolute;top:0;z-index:5;width:100%;height:100%;width:50%;border-right:2px solid #415671;}",
                ".title.jsx-1727865174{padding-top:80px;padding-bottom:60px;font-weight:300;text-transform:capitalize;font-size:22px;font-family:adobe-garamond-pro, serif;}",
                ".pre-title.jsx-1727865174{display:block;text-transform:uppercase;font-weight:bold;-webkit-letter-spacing:2px;-moz-letter-spacing:2px;-ms-letter-spacing:2px;letter-spacing:2px;font-size:10px;margin-bottom:20px;font-family:brandon-grotesque, sans-serif;}",
                "@media screen and (min-width: 768px){.root.jsx-1727865174{padding-top:120px;}.title.jsx-1727865174{font-size:36px;}}",
                "@media screen and (min-width: 1024px){.title.jsx-1727865174{font-size:44px;padding-bottom:120px;}.pre-title.jsx-1727865174{margin-bottom:50px;}}",
                "@media screen and (min-width: 1366px){.root.jsx-1727865174{padding-top:160px;}.title.jsx-1727865174{font-size:42px;}}",
              ]
            }
            styleId="1727865174"
          />,
        ],
        "className": "jsx-1727865174 root",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "jsx-1727865174 center-line",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "breakpoints": Array [
              "screen and (min-width: 768px)",
              "screen and (min-width: 1024px)",
              "screen and (min-width: 1366px)",
            ],
            "children": Array [
              <Medallion
                dimension={70}
                iconURL="foo.com"
                theme={
                  Object {
                    "left": "50%",
                    "position": "absolute",
                    "top": "-30px",
                    "transform": "translateX(-50%)",
                    "zIndex": "20",
                  }
                }
              />,
              <HR
                theme={
                  Object {
                    "borderTop": "2px solid #c2a068",
                    "top": "6px",
                  }
                }
              />,
              <HR
                theme={
                  Object {
                    "borderTop": "1px solid #e2b979",
                    "top": "9px",
                  }
                }
              />,
              <h2
                className="jsx-1727865174 title"
              >
                <span
                  className="jsx-1727865174 pre-title"
                >
                  A guide to
                </span>
                The solar system
              </h2>,
            ],
            "theme": Object {
              "background": "url(https://vega.slooh.com/assets/v4/common/White_Paper_Subtle.png)",
              "boxShadow": "rgb(191, 191, 191) 0px 11px 20px -10px",
              "position": "relative",
              "zIndex": "10",
            },
            "widths": Array [
              "620px",
              "780px",
              "940px",
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "dimension": 70,
                "iconURL": "foo.com",
                "theme": Object {
                  "left": "50%",
                  "position": "absolute",
                  "top": "-30px",
                  "transform": "translateX(-50%)",
                  "zIndex": "20",
                },
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "theme": Object {
                  "borderTop": "2px solid #c2a068",
                  "top": "6px",
                },
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "theme": Object {
                  "borderTop": "1px solid #e2b979",
                  "top": "9px",
                },
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <span
                    className="jsx-1727865174 pre-title"
                  >
                    A guide to
                  </span>,
                  "The solar system",
                ],
                "className": "jsx-1727865174 title",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "A guide to",
                    "className": "jsx-1727865174 pre-title",
                  },
                  "ref": null,
                  "rendered": "A guide to",
                  "type": "span",
                },
                "The solar system",
              ],
              "type": "h2",
            },
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "css": Array [
              ".root.jsx-1727865174{position:relative;text-align:center;font-family:brandon-grotesque, sans-serif;color:#415671;background:url(https://vega.slooh.com/assets/v4/common/Dark_Pattern_Featured_LargeCanvas.png);padding-top:70px;}",
              ".center-line.jsx-1727865174{position:absolute;top:0;z-index:5;width:100%;height:100%;width:50%;border-right:2px solid #415671;}",
              ".title.jsx-1727865174{padding-top:80px;padding-bottom:60px;font-weight:300;text-transform:capitalize;font-size:22px;font-family:adobe-garamond-pro, serif;}",
              ".pre-title.jsx-1727865174{display:block;text-transform:uppercase;font-weight:bold;-webkit-letter-spacing:2px;-moz-letter-spacing:2px;-ms-letter-spacing:2px;letter-spacing:2px;font-size:10px;margin-bottom:20px;font-family:brandon-grotesque, sans-serif;}",
              "@media screen and (min-width: 768px){.root.jsx-1727865174{padding-top:120px;}.title.jsx-1727865174{font-size:36px;}}",
              "@media screen and (min-width: 1024px){.title.jsx-1727865174{font-size:44px;padding-bottom:120px;}.pre-title.jsx-1727865174{margin-bottom:50px;}}",
              "@media screen and (min-width: 1366px){.root.jsx-1727865174{padding-top:160px;}.title.jsx-1727865174{font-size:42px;}}",
            ],
            "styleId": "1727865174",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
